docs/source/conf.py

```diff
@@ -70,7 +79,6 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
```

Comment:
Can you either move the other code block down, or the comment above this one up? Now the comment doesn't make sense anymore